tools/eslint-rules/inspector-check.js

```diff
@@ -30,8 +30,10 @@ module.exports = function(context) {
   }
 
   function reportIfMissing(context, node) {
-    if (usesInspector && !hasInspectorCheck) {
-      context.report(node, msg);
+    if (missingCheckNodes.length > 0 && !hasInspectorCheck) {
```

Comment:
You don't have to check `missingCheckNodes.length > 0`, just the `!hasInspectorCheck` is enough.

Reply:
Ah right, I'll fix that. Thanks